acceptance/gossip_peerings_test.go

```diff
@@ -83,14 +107,14 @@ func TestGossipPeerings(t *testing.T) {
 	l.Start()
 	defer l.Stop()
 
-	checkGossipPeerings(t, l, 20*time.Second)
+	checkGossip(t, l, 20*time.Second, hasPeers(len(l.Nodes)))
```

Comment:
Yeah, it's a little repetitive but I don't think it's bad enough to pull out and name. And we will presumably have other tests where the number of peers is not constant.